bootstrapvz/plugins/apt_proxy/tasks.py

```diff
@@ -34,11 +34,21 @@ class SetAptProxy(Task):
 	@classmethod
 	def run(cls, info):
 		proxy_path = os.path.join(info.root, 'etc/apt/apt.conf.d/02proxy')
+		proxy_username = info.manifest.plugins['apt_proxy'].get('username')
```

Comment:
You will get a `KeyError` here if you do not supply a `None` as the default in the second parameter.